app/src/main/java/tech/relaycorp/letro/data/entity/AccountDataModel.kt

```diff
@@ -0,0 +1,14 @@
+package tech.relaycorp.letro.data.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+const val USER_TABLE_NAME = "user"
+
+@Entity(tableName = USER_TABLE_NAME)
+data class AccountDataModel(
+    @PrimaryKey
+    val nodeId: String,
+    val username: String,
```

Comment:
Is this really the username (e.g., `alice`) or the id (e.g., `alice@example.com`)?

Also, this should unique too.

Reply:
Changed to address and made it as a PrimaryKey. Removed nodeId as it's saved in the DataStore